src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Providers.OSUI.RangeSlider.NoUISlider.Utils {
+	export function SetRangeValues(providerConfigs: ProviderConfigs): ProviderConfigs {
```

Comment:
I think there might be a Flatpickr.Range type, instead ProviderConfigs

Reply:
I'm using the same type as the NormalizeProviderConfigs method. I'm passing all the configs and not only one.